spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/cql/CassandraConnectionFactory.scala

```diff
@@ -73,9 +96,36 @@ object DefaultConnectionFactory extends CassandraConnectionFactory {
       .withSocketOptions(options)
   }
 
+  /** Returns the Cluster.Builder object used to setup Cluster instance. */
+  def clusterSSLBuilder(conf: CassandraConnectorConf, builder: Cluster.Builder): Cluster.Builder = conf.sslEnabled match {
+    case false => builder
+    case true => clusterSSLBuilderOptions(conf, builder)
+  }
+
+  private def clusterSSLBuilderOptions(conf: CassandraConnectorConf, builder: Cluster.Builder): Cluster.Builder = conf.sslTrustStorePath match {
+    case None => builder.withSSL()
+    case path: Some[String] => {
+      val trustStoreFile = new FileInputStream(path.get)
+      val context = SSLContext.getInstance("SSL")
```

Comment:
we could.. I decided to keep it simple cause of time constrains.
Do you feel this needs to be configurable?
You can always convert your keystores to JKS (like it is required with most Netflix projects).